docs/configure-archive-logs.md

```diff
@@ -59,3 +61,25 @@ spec:
     archiveLocation:
       archiveLogs: true
 ```
+
+## Suggested alternatives
+
+Argo Workflows log storage facilities are quite basic.
+
+We recommend using a fully featured Kubernetes aware logging facility which will provide you with facilities for indexing, searching and managing of log storage.
+
+We also recommend adding [links](links.md) to connect from the Argo UI to your logging facility's UI. Examples are given in the [workflow-controller-configmap.yaml](workflow-controller-configmap.yaml).
+
+* Link `scope: workflow` to the logs of a Workflow
+* Link `scope: pod-logs` to the logs of a specific Pod of a Workflow
+* Parametrize the link with `${metadata.name}`, `${metadata.namespace}`, `${metadata.labels}`, and other available metadata
+
+We do not plan to reach feature parity with purpose built-logging facilities within workflows as there are more optimized tools available in the ecosystem. Some open-source products which are known to work and you could consider are
+
+* [`fluentd`](https://github.com/fluent/fluentd) for collection
+* [ELK](https://www.elastic.co/elastic-stack/) as storage, querying and a UI
+* [`promtail`](https://grafana.com/docs/loki/latest/send-data/promtail/) for collection
+* [`loki`](https://grafana.com/docs/loki/latest/) for storage and querying
+* [`grafana`](https://grafana.com/docs/grafana/latest/) for a UI
+
+Almost any logging system which integrates with kubernetes should be able to be used however.
```

Comment:
I don't think this needs to be said as we already mention k8s aware logging facilities multiple times. With the change above, it also says "include:" and so does not exclude others.
```suggestion
```

If you really want to leave this in (simplicity is preferred per the k8s style guide), I'd still have suggestions:
```suggestion
Any other Kubernetes-aware logging system can also be used.
```
- simpler / more direct and improved tense, per [k8s style guide](https://kubernetes.io/docs/contribute/style/style-guide/#content-best-practices)
  - General note: "hedging" wording runs counter to the directness and tense directions. I get the uncertainty as I use similar wording too, but for docs the purpose & audience is a bit different. And yes, not everything may work per se, but that is out of scope (and tbh implied for all software as a whole).
- consistently use "Kubernetes-aware"

Reply:
Done